src/modules/ROOT/pages/changelog/202309/00.json

```diff
@@ -0,0 +1,261 @@
+{
+  "meta": {
+    "releasedOn": "2023.11.07"
+  },
+
+  "added": [
+    {
+      "what": [
+        "a btn:[Back] to navigate to previous page block modal when viewing or editing a tabbed page block."
+      ],
+      "why": [
+        "to provide the user with an easier way to go back in a modal."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1448"
+      ]
+    },
+    {
+      "what": [
+        "dropdown with values `25`, `50`, `100` in the record list configurator to set the amount of records that can be loaded per page in a record list.",
```

Comment:
How is that different from limit? Worth explaining because they sound the same to me